test/test_datelist.py

```diff
@@ -0,0 +1,47 @@
+import os
```

Comment:
Do we have a test for all date parsing option? Date; date1-date2; date1-date2-sampling?